### PR TITLE
VyOS: Add vyos_static_routes model

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>
VyOS static routes model

### CODE REVIEW

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+              description:
+                - ip address of next-hop router.
+              type: list
+              elements: dict
+              suboptions:
+                address:
+                  description: IPv4/v6 address of the next hop.
+                  type: str
+                  required: True
+                enable:
```
Comment (trishnaguha) [inline models/vyos/static_routes/vyos_static_route.yaml]: Should be `enabled`.

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+NETWORK_OS: vyos
+RESOURCE: static_routes
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: vyos_static_routes
+  version_added: 2.10
+  short_description: Manages attributes of static routes on VyOS network devices.
```
Comment (samccann) [inline models/vyos/static_routes/vyos_static_route.yaml]:
```suggestion
  short_description: Manages attributes of static routes on VyOS network devices
```

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            - Configured to silently discard packets.
+          type: dict
+          suboptions:
+            distance:
+              description:
+                - Distance for the route.
+              type: int
+        next_hop:
+          description:
+            - ip address of next-hop router.
```
Comment (samccann) [inline models/vyos/static_routes/vyos_static_route.yaml]:
```suggestion
            - IP address of next-hop router.
```

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+          type: list
+          elements: dict
+          suboptions:
+            afi:
+              description:
+                - Specifies the top level address family indicator.
+              type: str
+              choices: ['ipv4', 'ipv6']
+              required: True
+        routes:
```
Comment (NilashishC) [inline models/vyos/static_routes/vyos_static_route.yaml]: `routes` should be a child of `address_families`

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                description:
+                  - Distance for the route.
+                type: int
+          next_hops:
+            description:
+              - IP address of next-hop router.
+            type: list
+            elements: dict
+            suboptions:
+              address:
```
Comment (NilashishC) [inline models/vyos/static_routes/vyos_static_route.yaml]: This should be `forward_router_address`

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+                description:
+                  - IPv4/v6 address of the next hop.
+                type: str
+                required: True
+              enabled:
+                description:
+                  - Disable IPv4/v6 next-hop static route.
+                type: bool
+              distance:
```
Comment (NilashishC) [inline models/vyos/static_routes/vyos_static_route.yaml]:
```suggestion
              admin_distance:
```

File: models/vyos/static_routes/vyos_static_route.yaml [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+                description:
+                  - Configured to discard packets.
+                type: bool
+              distance:
+                description:
+                  - Distance for the route.
+                type: int
+          next_hops:
+            description:
+              - IP address of next-hop router.
```
Comment (NilashishC) [inline models/vyos/static_routes/vyos_static_route.yaml]: The description needs to be updated, I believe.

**Pull request merged**